pootle/apps/accounts/utils.py

```diff
@@ -30,3 +34,177 @@ def get_user_by_email(email):
             return User.objects.get(email__iexact=email)
         except User.DoesNotExist:
             return None
+
+
+def merge_user(src_user, target_user):
+    """Merges one user to another.
+
+    The following are fields are updated (model: fields):
+    - units: submitted_by, commented_by, reviewed_by
+    - submissions: submitter
+    - suggestions: user, reviewer
+
+    :param src_user: `User` instance to merge from.
+    :param target_user: `User` instance to merge to.
+    """
+
+    # Move units to target_user.
+    for unit in src_user.submitted.iterator():
+        unit.submitted_by = target_user
+        unit.save()
+
+    for unit in src_user.commented.iterator():
+        unit.commented_by = target_user
+        unit.save()
+
+    for unit in src_user.reviewed.iterator():
+        unit.reviewed_by = target_user
+        unit.save()
+
+    # Move submissions to target_user.
+    for submission in src_user.submission_set.iterator():
+        submission.submitter = target_user
+
+        # Before we can save we first have to remove existing score_logs for
+        # this submission - they will be recreated on save with correct user.
+        for score_log in submission.scorelog_set.iterator():
+            score_log.delete()
+        submission.save()
+
+    # Move suggestions to target_user.
+    for suggestion in src_user.suggestions.iterator():
+        suggestion.user = target_user
+        suggestion.save()
+
+    for suggestion in src_user.reviews.iterator():
+        suggestion.reviewer = target_user
+        suggestion.save()
+
+
+def remove_units_created_by(user):
+    """Remove units created by user that have not had further
+    activity.
+
+    :param user: `User` to remove units for.
+    """
+    # Delete units created by user without submissions by others.
+    created_by_user = (user.submission_set
+                       .filter(type=SubmissionTypes.UNIT_CREATE))
```

Comment:
Are the parens to avoid a backslash for the line wrap? We are not using this style anywhere with querysets ([see example](https://github.com/translate/pootle/blob/56e70821d8865e441a34c4dfb546ccfcad228568/pootle/apps/pootle_store/models.py#L825-L826)). Either way, please align the `.filter()` vertically. Same applies to elsewhere in this PR.